packages/yoroi-extension/app/themes/skins/FormFieldOwnSkin.js

```diff
@@ -76,8 +76,8 @@ export const FormFieldOwnSkin = class extends React.Component<Props, State> {
             {(this.props.type === 'password') ? (
               <button tabIndex="-1" type="button" onClick={this.showPassword}>
                 {isPasswordShown
-                  ? <PasswordSvg />
-                  : <PasswordHiddenSvg />}
+                  ? <PasswordHiddenSvg />
+                  : <PasswordSvg />}
```

Comment:
Let' also rename the icons while we are here, to more consistent and readable: `PasswordShownIcon` and `PasswordHiddenIcon`